build/Packages.json

```
@@ -1,10 +1,10 @@
 {
     "Microsoft.Dynamics.BusinessCentral.Translations":  {
-                                                            "Version":  "24.0.20240220.5",
+                                                            "Version":  "24.1.20240227.4",
                                                             "Source":  "NuGet.org"
                                                         },
     "AppBaselines-BCArtifacts":  {
-                                     "Version":  "23.4.15643.16362",
+                                     "Version":  "24.0.16561.0",

```

```suggestion
                                     "Version":  "24.0.16410.16595",

```

Latest artifact out of 24.0